math/src/transcendental.rs

```diff
@@ -428,12 +429,12 @@ mod tests {
 
 		assert_eq!(
 			pow(S::from_num(22.1234), S::from_num(2.1)),
-			Ok(D::from_num(667.096912176457))
+			Ok(D::from_str("667.0969121771803182631954923946").unwrap())
```

Comment:
wolfram result : 667.09691217718012362888000511134534475469139572779466865919

Reply:
looks to me that is more accurate than before